README.md

```diff
@@ -1,2 +1,570 @@
-# predict-credit-risk-with-jupyter-on-cloud-pak-for-data
-Use IBM Cloud Pak for Data and Jupyter to predict credit risk
+# Predict Credit Risk With Jupyter on Cloud Pak for data
+
+In this Code Pattern, we will go through the process of exploring a data set and building a predictive model that can be used to determine the likelyhood of a credit loan having 'Risk' or 'No Risk'. For this use case, the machine learning model we are building is a classification model that will return a prediction of 'Risk' (the features of the loan applicant predict that there is a good chance of default on the loan) or No Risk (the applicant's inputs predict that the loan will be paid off). The approach we will take in this lab is to some fairly popular libraries / frameworks to build the model in Python using a Jupyter notebook. Once we have built the model, we will make it available for deployment so that it can be used by others.
+
+After building the model, we will go through the process of deploying a machine learning model so it can be used by others. Deploying a model allows us to put a model into production, so that data can be passed to it to return a prediction. The deployment will result in an endpoint that makes the model available for wider use in applications and to make business decisions. There are several types of deployments available ([depending on the model framework used](https://www.ibm.com/support/producthub/icpdata/docs/content/SSQNUZ_current/wsj/analyze-data/pm_service_supported_frameworks.html)), of which we will explore:
```

Comment:
first sentence is a repeat of previous sentence

Reply:
fixed, thanks.